CMakeLists.txt

```diff
@@ -10,8 +10,9 @@ include(cmake/install-prefix.cmake)
 include(cmake/wsl.cmake)
 include(cmake/simd.cmake)
 
-set(CMAKE_C_STANDARD 11)
 set(CMAKE_CXX_STANDARD 20)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
This flag is set to OFF by default, which means that if the compiler doesn't support the standard, it will fall back to the latest standard supported by the compiler. Turning this flag ON prevents the fallback behavior.